urllib3/response.py

```diff
@@ -387,48 +386,67 @@ def readinto(self, b):
             b[:len(temp)] = temp
             return len(temp)
 
-    def read_chunked(self, amt=None):
+    def _get_next_chunk(self):
+        # First, we'll figure out length of a chunk and then
+        # we'll try to read it from socket.
+        if self.chunk_left is None:
+            line = self._fp.fp.readline()
+            line = line.decode()
```

Comment:
What's this decode call doing here?

I missed it in the last code review, but it looks wrong.